ddhcpd/files/lib/gluon/upgrade/400-ddhcpd-firewall

```diff
@@ -1,19 +1,19 @@
 #!/usr/bin/lua
 local e=require('simple-uci').cursor()
 e:section('firewall','rule','mesh_ddhcpd',{
-name='mesh_ddhcpd',
-src='mesh',
-family='ipv6',
-dest_port='1234',
-proto='udp',
-target='ACCEPT',
+  name='mesh_ddhcpd',
+  src='mesh',
+  family='ipv6',
+  dest_port='1234',
+  proto='udp',
+  target='ACCEPT',
```

Comment:
There is a reason why there are no indents here.

Reply:
what reason?